src/connectedk8s/azext_connectedk8s/_params.py

```diff
@@ -0,0 +1,27 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long
+
+
+def load_arguments(self, _):
+
+    from azure.cli.core.commands.parameters import tags_type
+
+    with self.argument_context('connectedk8s connect') as c:
+        c.argument('tags', tags_type)
+        c.argument('cluster_name', options_list=['--name', '-n'], help='The name of the connected cluster.')
+        c.argument('kube_config', options_list=['--kube-config'], help='Path to the kube config file.')
+        c.argument('kube_context', options_list=['--kube-context'], help='Kubconfig context from current machine.')
+
+    with self.argument_context('connectedk8s list') as c:
+        pass
+
+    with self.argument_context('connectedk8s show') as c:
+        c.argument('cluster_name', options_list=['--name', '-n'], help='The name of the connected cluster.')
+
+    with self.argument_context('connectedk8s delete') as c:
+        c.argument('cluster_name', options_list=['--name', '-n'], help='The name of the connected cluster.')
+        c.argument('kube_config', options_list=['--kube-config'], help='Path to the kube config file.')
+        c.argument('kube_context', options_list=['--kube-context'], help='Kubconfig context from current machine.')
```

Comment:
add `id_part='name'` for cluster_name in `show` and `delete` to support `--ids`.